cmd/performance-profile-creator/cmd/root.go

```diff
@@ -55,45 +55,90 @@ type ProfileData struct {
 // ClusterData collects the cluster wide information, each mcp points to a list of ghw node handlers
 type ClusterData map[*machineconfigv1.MachineConfigPool][]*profilecreator.GHWHandler
 
-// rootCmd represents the base command when called without any subcommands
-var rootCmd = &cobra.Command{
-	Use:   "performance-profile-creator",
-	Short: "A tool that automates creation of Performance Profiles",
-	RunE: func(cmd *cobra.Command, args []string) error {
-		mustGatherDirPath := cmd.Flag("must-gather-dir-path").Value.String()
-		cluster, err := getClusterData(mustGatherDirPath)
-		if err != nil {
-			return fmt.Errorf("failed to parse the cluster data: %v", err)
-		}
+func init() {
+	log.SetOutput(os.Stderr)
+	log.SetFormatter(&log.TextFormatter{
+		DisableTimestamp: true,
+	})
+}
 
-		info, err := strconv.ParseBool(cmd.Flag("info").Value.String())
-		if err != nil {
-			return fmt.Errorf("failed to parse the info flag: %v", err)
-		}
-		if info {
-			showClusterData(cluster)
+// NewRootCommand returns entrypoint command to interact with all other commands
+func NewRootCommand() *cobra.Command {
+	pcArgs := &ProfileCreatorArgs{
+		UserLevelNetworking: pointer.BoolPtr(false),
+	}
+
+	var requiredFlags []string
+
+	root := &cobra.Command{
+		Use:   "performance-profile-creator",
+		Short: "A tool that automates creation of Performance Profiles",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if !cmd.Flag("must-gather-dir-path").Changed {
+				return fmt.Errorf("missing cluster data")
```

Comment:
Lets make this more verbose and let the user know that the ` --must-gather-dir-path` must be specified.

Reply:
Ack, done.